src/pages/AppHub/AppHub.js

```diff
@@ -0,0 +1,151 @@
+import { useDataQuery } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import { PropTypes } from '@dhis2/prop-types'
+import { InputField, Pagination } from '@dhis2/ui'
+import { NoticeBox, CenteredContent, CircularLoader } from '@dhis2/ui'
+import React, { useEffect } from 'react'
+import { useHistory } from 'react-router-dom'
+import { useDebounce } from 'use-debounce'
+import {
+    useQueryParams,
+    StringParam,
+    NumberParam,
+    withDefault,
+} from 'use-query-params'
+import { AppCard } from '../../components/AppCard/AppCard'
+import { AppCards as AppCards_ } from '../../components/AppCards/AppCards'
+import { getLatestVersion } from '../../get-latest-version'
+import styles from './AppHub.module.css'
+
+const query = {
+    appHub: {
+        resource: 'appHub/v2/apps',
+        params: params => ({
```

Comment:
The DHIS version should be passed as a parameter so that only compatible apps are shown.